cmd/cassandra-reader/main.go

```diff
@@ -6,312 +6,107 @@ package main
-	}
-
-	conn, err := grpc.Dial(cfg.usersAuthURL, opts...)
-	if err != nil {
-		logger.Error(fmt.Sprintf("Failed to connect to auth service: %s", err))
-		os.Exit(1)
-	}
-	logger.Info(fmt.Sprintf("Established gRPC connection to things via gRPC: %s", cfg.usersAuthURL))
-	return conn
-}
-
-func loadConfig() config {
-	dbPort, err := strconv.Atoi(mainflux.Env(envDBPort, defDBPort))
-	if err != nil {
-		log.Fatal(err)
+	// create cassandra reader service configurations
+	cfg := config{}
+	// load cassandra reader service configurations from environment
+	if err := env.Parse(&cfg); err != nil {
+		log.Fatalf("failed to load %s service configuration : %s", svcName, err.Error())
 	}
 
-	dbCfg := cassandra.DBConfig{
-		Hosts:    strings.Split(mainflux.Env(envCluster, defCluster), sep),
-		Keyspace: mainflux.Env(envKeyspace, defKeyspace),
-		User:     mainflux.Env(envDBUser, defDBUser),
-		Pass:     mainflux.Env(envDBPass, defDBPass),
-		Port:     dbPort,
-	}
-
-	tls, err := strconv.ParseBool(mainflux.Env(envClientTLS, defClientTLS))
+	// create new logger
+	logger, err := logger.New(os.Stdout, cfg.LogLevel)
 	if err != nil {
-		log.Fatalf("Invalid value passed for %s\n", envClientTLS)
+		log.Fatalf(err.Error())
 	}
 
-	authTimeout, err := time.ParseDuration(mainflux.Env(envThingsAuthTimeout, defThingsAuthTimeout))
+	// create new thing grpc client
+	tc, tcHandler, err := thingsClient.Setup(envPrefix, cfg.JaegerURL)
 	if err != nil {
-		log.Fatalf("Invalid %s value: %s", envThingsAuthTimeout, err.Error())
+		log.Fatal(err.Error())
 	}
+	defer tcHandler.Close()
+	logger.Info("Successfully connected to things grpc server " + tcHandler.Secure())
 
-	usersAuthTimeout, err := time.ParseDuration(mainflux.Env(envUsersAuthTimeout, defUsersAuthTimeout))
+	// create new auth grpc client
+	auth, authHandler, err := authClient.Setup(envPrefix, cfg.JaegerURL)
 	if err != nil {
-		log.Fatalf("Invalid %s value: %s", envThingsAuthTimeout, err.Error())
-	}
-
-	return config{
-		logLevel:          mainflux.Env(envLogLevel, defLogLevel),
-		port:              mainflux.Env(envPort, defPort),
-		dbCfg:             dbCfg,
-		clientTLS:         tls,
-		caCerts:           mainflux.Env(envCACerts, defCACerts),
-		serverCert:        mainflux.Env(envServerCert, defServerCert),
-		serverKey:         mainflux.Env(envServerKey, defServerKey),
-		jaegerURL:         mainflux.Env(envJaegerURL, defJaegerURL),
-		thingsAuthURL:     mainflux.Env(envThingsAuthURL, defThingsAuthURL),
-		usersAuthURL:      mainflux.Env(envUsersAuthURL, defUsersAuthURL),
-		usersAuthTimeout:  usersAuthTimeout,
-		thingsAuthTimeout: authTimeout,
+		log.Fatal(err.Error())
 	}
-}
+	defer authHandler.Close()
+	logger.Info("Successfully connected to auth grpc server " + authHandler.Secure())
 
-func connectToCassandra(dbCfg cassandra.DBConfig, logger logger.Logger) *gocql.Session {
-	session, err := cassandra.Connect(dbCfg)
+	////////// CASSANDRA READER REPO /////////////
```

Comment:
No need for this comment, keep the more discrete

Reply:
Maybe just `// Cassandra reader repo` with the blank line below, or some other more aligned form.

Reply:
Done